### DISCUSSION TITLE
表单标题内联

### DISCUSSION THREAD

**Discussion by WT-Road** [ant-design/ant-design#46041] [Q&A] [answered]
Antd怎么实现和谷歌登录一样，表单的标题放在输入框中，输入或者有值时标题上移
![a5833752-0376-48e1-ab34-89afcaabc92c](https://github.com/ant-design/ant-design/assets/14371925/a9157555-370d-434e-b0c4-177ed8d34ed9)
![d6d0a83b-1866-4864-ac5f-a66fee050c48](https://github.com/ant-design/ant-design/assets/14371925/77173bea-c4ab-4ab0-8354-7ee0a2125bdc)

**Comment by WT-Road** [comment]
https://v4.mui.com/zh/components/text-fields/

**Comment by Wxh16144** [comment]
antd 貌似不支持，得自己搞了。

**Reply by WT-Road** [reply]
https://codepen.io/yogwang/pen/XvYEaJ
思路是将label当做标题，但是label和输入框不在同一个层级，没法监听是否有值，同级的话到是可以

**Comment by afc163** [comment, marked as answer]
See https://github.com/ant-design/ant-design/issues/16323
